src/redux/uniqueTokens.js

```diff
@@ -150,7 +151,8 @@ export const fetchUniqueTokens = showcaseAddress => async (
     }
   };
 
-  fetchPage();
+  fetchPage(network);
+  fetchPage(AssetType.polygon);
```

Comment:
One thing we should consider is to not parallelize both requests, cause we could get rate limited.

Might be better if we fetch polygon shit after we're done with mainnet